db/migrate/20220613210355_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,11 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[7.0]
+  def change
+    change_table(:users) do |t|
+      t.string :confirmation_token 
+      t.datetime :confirmed_at 
+      t.datetime :confirmation_sent_at 
+      t.string :unconfirmed_email
+    end
+    add_index :users, :confirmation_token, :unique => true 
+  end
+end
```

Comment:
# OPTIONAL

- [ ] Great job on adding the confirmation on devise however you have this is already in the main devise migration(`db/migrate/20220613193104_add_devise_to_users.rb`) what you just need to do is to uncomment it.

![image](https://user-images.githubusercontent.com/65068771/173458443-7891e39c-a7c3-4c2e-9faf-81e671fb3511.png)